example/cluster-machine-template-simple.yaml

```diff
@@ -39,16 +39,14 @@ spec:
     net:
       name: cluster-api-net
       clusterName: cluster-api
-      ipRange: "10.0.0.0/24"
+      ipRange: "10.0.0.0/16"
```

Comment:
Please change to use the new default values.

Reply:
The default net Ip range is 10.0.0.0/16

Reply:
Ok all good